cdc/kv/client.go

```diff
@@ -851,6 +851,7 @@ func (s *eventFeedSession) requestRegionToStore(
 
 			limiter := s.client.getRegionLimiter(regionID)
 			g.Go(func() error {
+				defer s.deleteStream(rpcCtx.Addr)
 				if !s.enableKVClientV2 {
 					return s.receiveFromStream(ctx, g, rpcCtx.Addr, getStoreID(rpcCtx), stream, pendingRegions, limiter)
```

Comment:
```suggestion
					return s.receiveFromStream(streamCtx, g, rpcCtx.Addr, getStoreID(rpcCtx), stream, pendingRegions, limiter)
```

What about streamCtx?

Reply:
`streamCtx` is used for gRPC stream only, reuse it in `receiveFromStream` could cause unexpected context cancel detection.